guides/v2.3/graphql/reference/cms.md

```diff
@@ -0,0 +1,128 @@
+---
+group: graphql
+title: CMS endpoints
+---
+
+The `cmsPage` and `cmsBlock` endpoints return information about content pages and blocks that were developed with the Magento Content Management System (CMS).
+
+## Query structures
+
+Return the contents of a CMS page:
+
+`cmsPage(id: Int): CmsPage`
+
+Return the contents of one or more CMS blocks:
+
+`cmsBlocks(identifiers: [String]): CmsBlocks`
+
+### Input attributes
+
+Attribute | Data type | Description
```

Comment:
Should all tables have the `{:style="table-layout:auto;"}` styling?

Reply:
Missed that one. Updated.